ios/brave-ios/Sources/BraveUI/SwiftUI/URLElidedTextView.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveCore
```

Comment:
`BraveUI` doesn't have a dependency on `BraveCore` and probably shouldn't, maybe better to keep this in BraveShared despite it being UI